### PR TITLE
Add more fields to `GET /node/consensus` response

### PR DESCRIPTION
**Pull request by jumaffre**
This is so operators can determine the triplet `{Raft state, current view, primary id}` by querying only one endpoint (see #3596).

### CODE REVIEW

File: src/consensus/aft/impl/state.h [hunk 1]
```diff
@@ -141,7 +141,6 @@ namespace aft
     State(const ccf::NodeId& my_node_id_) : my_node_id(my_node_id_) {}
 
     std::mutex lock;
-    std::map<ccf::NodeId, std::shared_ptr<Replica>> configuration;
```
Comment (jumaffre) [inline src/consensus/aft/impl/state.h:144]: This was unused.

**Review by eddyashton — APPROVED** [review]

**Comment by jumaffre** [comment]
Merging this as the Daily failure is unrelated and has already been fixed.

**Pull request merged**